sound/soc/codecs/rt715-sdw-sdca.c

```diff
@@ -0,0 +1,479 @@
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_MUTE_CTRL, CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_MUTE_CTRL, CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_MUTE_CTRL, CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_MUTE_CTRL, CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_MUTE_CTRL, CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_MUTE_CTRL, CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_VOL_CTRL, CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_VOL_CTRL, CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_VOL_CTRL, CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC8_9_VOL, FU_VOL_CTRL, CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, SMPU_TRIG_ST_EN, SMPU_TRIG_EN_CTRL,
+		CH_00):
+	case RT715_SDCA_ADDR(FUNC_02, SMPU_TRIG_ST_EN, SMPU_TRIG_ST_CTRL,
+		CH_00):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_VOL_CTRL, CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_VOL_CTRL, CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_VOL_CTRL, CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC10_11_VOL, FU_VOL_CTRL, CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC7_27_VOL, FU_MUTE_CTRL, CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC7_27_VOL, FU_MUTE_CTRL, CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC7_27_VOL, FU_VOL_CTRL, CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_ADC7_27_VOL, FU_VOL_CTRL, CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_05):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_06):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_07):
+	case RT715_SDCA_ADDR(FUNC_02, FU_AMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_08):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_01):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_02):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_03):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_04):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_05):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_06):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_07):
+	case RT715_SDCA_ADDR(FUNC_02, FU_DMIC_GAIN_EN, FU_DMIC_GAIN_CTRL,
+		CH_08):
+		regmap_write(rt715->sdw_regmap,
+			reg | BIT(13), ((val >> 8) & 0xff));
+		regmap_write(rt715->sdw_regmap,
+			reg, (val & 0xff));
+		goto _done_;
+	default:
+		break;
+	}
+
+	/* SDCA mapping case or normal address */
+	if (reg & 0x40000000 || reg < 0xffff) {
+		regmap_write(rt715->sdw_regmap,
+			reg, val);
+	}
```

Comment:
superfluous braces. You can also fit the `regmap_write()` onto one line, there were a couple more similar unnecessary line-breaks above